ml-agents-envs/mlagents_envs/mock_communicator.py

```diff
@@ -50,7 +50,8 @@ def initialize(self, inputs: UnityInputProto) -> UnityOutputProto:
         )
         rl_init = UnityRLInitializationOutputProto(
             name="RealFakeAcademy",
-            version=UnityEnvironment.API_VERSION,
+            communication_version=UnityEnvironment.API_VERSION,
+            package_version="0.42.0-preview",
```

Comment:
can we make it 42000 just to make sure we will not collide?
```suggestion
            package_version="mock_package_version",
```